docs/en-US/component/message.md

```diff
@@ -121,7 +121,7 @@ ElMessage({}, appContext)
 ### Options
 
 | Name                     | Description                                                                                          | Type                                                 | Default |
-|--------------------------|------------------------------------------------------------------------------------------------------|------------------------------------------------------|---------|
+| ------------------------ | ---------------------------------------------------------------------------------------------------- | ---------------------------------------------------- | ------- |
```

Comment:
Why my idea format style is different from yours?
![image](https://github.com/element-plus/element-plus/assets/38392315/309c20c4-6f6d-4846-a03e-19712bff1955)

Reply:
Do you have any special configuration? There are spaces between the dividing lines of other components. 🤔

Reply:
I use webstorm, may be different.

Reply:
This may be the reason.